extide/gradle/src/org/netbeans/modules/gradle/nodes/ConfigurationsNode.java

```diff
@@ -134,8 +134,12 @@ public Image getOpenedIcon(int type) {
     })
     @Override
     public String getShortDescription() {
-        GradleProject gp = project.getGradleProject();
-        return gp.getBaseProject().isResolved() ? Bundle.HINT_ConfigurationsNode() : Bundle.HINT_ConfigurationsNodeUnresolved();
+        return !needsResolve() ? Bundle.HINT_ConfigurationsNode() : Bundle.HINT_ConfigurationsNodeUnresolved();
+    }
+
+    private boolean needsResolve() {
+        GradleBaseProject gbp = GradleBaseProject.get(project);
+        return !gbp.isResolved() && !gbp.hasPlugins("java-platform"); //NOI18N
```

Comment:
Hm, this means that if a configuration is NOT resolved , but the project does use `java-platform` plugin - the project  as a whole ceases needing to be resolved ?

Reply:
Well it is about the warning badge and the tooltip. We put warning the configuration is not resolved in general.
However java-platform project can't be resolved by definition. So we do not place warning badge and unresolved config hint on the node.